features/steps/thens/alignment.py

```diff
@@ -110,3 +127,43 @@ def step_impl(context, inst, ifc_rep_criteria, ifc_layout_criteria):
                                 yield ValidationOutcome(inst=inst, expected=ifc_layout_criteria,
                                                         observed="IfcAlignmentCant",
                                                         severity=OutcomeSeverity.ERROR)
+
+
+@gherkin_ifc.step(
+    'The alignment\'s {layout_type} layout must have the same number of segments as the shape representation')
+def step_impl(context, inst, layout_type):
```

Comment:
```suggestion
def step_impl(context, inst):
```
and remove `layout_type`